README.md

```diff
@@ -35,13 +35,13 @@ export const store = createStore([increment])
 
 // your NgModule
 
-import { NgStoreonModule } from 'ng-storeon';
+import { NgStoreonModule, STOREON } from 'ng-storeon';
```

Comment:
import { NgStoreonModule, STOREON } from '@storeon/angular';

Reply:
@irustm fixed and resolved conflict